### PR TITLE
Syntax error and confirm POC

### PR DESCRIPTION
**Pull request by loneil**
Signed-off-by: Lucas ONeil <lucasoneil@gmail.com>

Fix a merge issue in the main branch.
Move column to 'Actions' to match UX mockup
Try out PrimeVue confirm dialog, just a proof of concept test right now, final UX call to come later.

### CODE REVIEW

File: services/tenant-ui/frontend/src/components/issuance/Schemas.vue [hunk 1]
```diff
@@ -3,78 +3,47 @@
 
   <ProgressSpinner v-if="loading" />
   <div v-else>
-    <DataTable
```
Comment (Jsyro) [inline services/tenant-ui/frontend/src/components/issuance/Schemas.vue:6]: i think we like the formatter breaking these options into separate lines?

Comment (loneil) [reply services/tenant-ui/frontend/src/components/issuance/Schemas.vue:6]: Yeah I like it as well, looking into how we can get auto-formatter to pick up the settings properly though...

Comment (loneil) [reply services/tenant-ui/frontend/src/components/issuance/Schemas.vue:6]: Tracking ongoing formatter needs in https://apps.nrs.gov.bc.ca/int/jira/browse/TRACTION-401 can hopefully address next sprint ASAP

**Review by Jsyro — APPROVED** [review]

**Review by popkinj — APPROVED** [review]
Looks good.

**Pull request merged**